libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -162,4 +163,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://docs.novu.co/channels/sms/africas-talking',
     logoFileName: { light: 'africas-talking.svg', dark: 'africas-talking.svg' },
   },
+  {
+    id: SmsProviderIdEnum.Kavenegar,
+    displayName: `Kavenegar SMS`,
```

Comment:
```suggestion
    displayName: `Kavenegar`,
```
Considering the product name I think it is not needed to add `SMS` in the display name. Also the integration will be highlighting the channel, in this case, SMS.